src/responsibleai/rai_analyse/generate_pdf.py

```diff
@@ -0,0 +1,1226 @@
+import argparse
+import json
+import logging
+import os
+
+import plotly.graph_objects as go
+import base64
+import plotly.io as pio
+
+from domonic.html import *
```

Comment:
I take it that this is the source of all the `p()`, `div()` etc>?

Reply:
yep, cleaned up with explicit imports instead of *.